pico/abus.pio

```diff
@@ -25,7 +25,7 @@ next_bus_cycle:
     wait 1 GPIO, PHI0_GPIO              ; wait for PHI0 to rise. Data propagation through the transceiver should
                                         ; be complete by the time this happens.
     in PINS, 8                          ; read AddrHi[7:0]
-    set PINS, 0b101  [2]                ; enable AddrLo tranceiver and delay for transceiver propagation delay
+    set PINS, 0b101  [5]                ; enable AddrLo tranceiver and delay for transceiver propagation delay
```

Comment:
I have been working on a derivative of this project (see it at KFest next month!) and I wound up having to increase exactly this delay in order to achieve reliable timing, which I only figured out a few days ago.  Seeing this change made me feel somewhat vindicated, that it wasn't just me.

Reply:
Yes, I checked with an oscilloscope and noticed the pulse was to short. I guess depends what components you get plus how you route the signal in the pcb. Adding some extra room sure will help the majority of implementations.

Reply:
I am still using a breadboard so I was extremely suspect regarding my signals, and this is where I found my problem to be.  It makes sense as well- the data bus signals are getting the 2-cycle synchronization delay removed explictly.  This is on the assumption that they are stable when sampled.  But every time you change the 245 enables (like we are doing right here), they stop being stable, and need time to stabilize again before one can reliably sample them!

Reply:
Nice! I was wondering about what the difference could be too & maybe I just won the Pico & '245 silicon lotteries and they have tight timings.  Thanks for the extra data point and good luck with your card!